Description: Debian provides only pandoc-citeproc-0.4* and it builds happily with it.
Author: Dmitry Bogatov <KAction@gnu.org>
Forwarded: no
Last-Update: 2014-12-04
--- a/BlogLiterately.cabal
+++ b/BlogLiterately.cabal
@@ -60,7 +60,7 @@ Library
                    haxr >= 3000.9 && < 3000.11,
                    pandoc >= 1.12 && < 1.13,
                    pandoc-types >= 1.12 && < 1.13,
-                   pandoc-citeproc >= 0.1.2 && < 0.4,
+                   pandoc-citeproc >= 0.1.2,
                    highlighting-kate >= 0.5 && < 0.6,
                    data-default >= 0.5 && < 0.6,
                    lens >= 3.8 && < 4.3
